Sources/Hero.swift

```diff
@@ -62,6 +62,7 @@ public class Hero: HeroBaseController {
     }
   }
 
+  public var isAnimating:Bool = false
```

Comment:
I have tried use this transitioning, but the var maybe not accurate represent animation state.